### PR TITLE
Update userguide

### PR DESCRIPTION
**Pull request by dhlee03**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -350,15 +382,17 @@ Finds persons who contain any of the given keywords.
...
 **Examples:**
 
-- `find John` returns `john` and `John Doe`
-- `find alex david` returns `Alex Yeoh`, `David Li`<br>
+- `find John` includes `john` and `John Doe`
+- `find alex david` includes `Alex Yeoh`, `David Li`<br>
   ![img.png](images/findAlexDavid.png)
-- `find participant` returns all participants in the list
+- `find participant` includes all participants in the list
+- `find 1` includes the person with group number 1
```
Comment (bachletuankhai) [inline docs/UserGuide.md:395]: "includes" does not seem to be the right verb here. I suggest we keep the same verb as AB-3.

**Review by bachletuankhai — CHANGES_REQUESTED** [review]
Just a verb change. This will be our CS2101 UG too so we need to be careful about word choice.

File: docs/UserGuide.md (hunk 1 above)
Comment (dhlee03) [reply docs/UserGuide.md:395]: It was intentional as we support other information(email, comment,...) that was not supported in AB-3. the word participant might be included inside comment, so return is too restrictive word.

Comment (dhlee03) [reply docs/UserGuide.md:395]: I will change to "returns a list that includes"

Comment (bachletuankhai) [reply docs/UserGuide.md:395]: The name here refers to the whole contact with all of their information, not only name. I think your added examples also explain that.

**Review by bachletuankhai — APPROVED** [review]
LGTM

**Pull request merged**